docs/source/scripts/environment_besca.yml

```diff
@@ -1,188 +1,157 @@
-name: besca_2
+name: besca_22
```

Comment:
should be besca_2.2. If we need to avoid the dot for which ever reason we should consider doing besca_2-2 instead